Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -23,6 +23,7 @@ namespace Xamarin.Forms.Platform.MacOS
 {
 	public class LabelRenderer : ViewRenderer<Label, NativeLabel>
 	{
+		const int DefaultMinimunScaleFactor = 16;
```

Comment:
```suggestion
		const int DefaultMinimumScaleFactor = 16;
```

Reply:
Also, as [mentioned in the issue](https://github.com/xamarin/Xamarin.Forms/issues/7176#issuecomment-522444158), the default for Android is 12sp. Why don't we match that? Or, maybe even better, open this up for the end-user to set this themselves?

Reply:
Oh sorry! I read this issue and I thought it was 16... dooh 🤦‍♂️

> Or, maybe even better, open this up for the end-user to set this themselves?

Thats a good ideia!

Reply:
> Or, maybe even better, open this up for the end-user to set this themselves?

I think to call this property "AutoFitTextStep", IntelliSense will show this property associated with `AutoFitText`